File: crates/deslop/tests/fixtures/rust-merge-leafgap/pricing.rs
pub fn apply_standard(book: &mut Vec<(String, i64)>) {
    let label = "standard";
    let ceiling = 100;
    book.push((label.to_owned(), ceiling));
    book.push((label.to_uppercase(), ceiling * 2));
    book.push((label.to_lowercase(), ceiling + 7));
    book.push((label.trim().to_owned(), ceiling - 1));
    book.push((label.repeat(2), ceiling / 2));
    book.sort();
}

pub fn apply_premium(book: &mut Vec<(String, i64)>) {
    let label = "premium";
    let ceiling = 250;
    book.push((label.to_owned(), ceiling));
    book.push((label.to_uppercase(), ceiling * 2));
    book.push((label.to_lowercase(), ceiling + 7));
    book.push((label.trim().to_owned(), ceiling - 1));
    book.push((label.repeat(2), ceiling / 2));
    book.sort();
}

File: crates/deslop/tests/fixtures/dart-merge-leafgap/pricing.dart
void applyStandard(List<String> book) {
  final label = "standard";
  final ceiling = 100;
  book.add(label);
  book.add(label.toUpperCase());
  book.add(label.toLowerCase());
  book.add(label.trim());
  book.add(ceiling.toString());
  book.sort();
}

void applyPremium(List<String> book) {
  final label = "premium";
  final ceiling = 250;
  book.add(label);
  book.add(label.toUpperCase());
  book.add(label.toLowerCase());
  book.add(label.trim());
  book.add(ceiling.toString());
  book.sort();
}
